nft/handler.go

```diff
@@ -97,6 +98,44 @@ func (g grpcHandler) MintInvoiceUnpaidNFT(ctx context.Context, request *nftpb.NF
 	return g.MintNFT(ctx, mintReq)
 }
 
+// TokenTransfer will be called to transfer a token owned by the identity contract
+func (g grpcHandler) TokenTransfer(ctx context.Context, request *nftpb.TokenTransferRequest) (*nftpb.TokenTransferResponse, error) {
+	ctxHeader, err := contextutil.Context(ctx, g.config)
+	if err != nil {
+		apiLog.Error(err)
+		return nil, err
+	}
+
+	tokenID, err := TokenIDFromString(request.TokenId)
```

Comment:
Why export this function ?

Reply:
We are currently using it in the integration tests